spec/decorators/ephemera_folder_decorator_spec.rb

```diff
@@ -70,6 +70,14 @@
     end
   end
 
+  context 'Will not error with no collections' do
+    let(:collection) { FactoryBot.create_for_repository(:collection) }
+    let(:resource) { FactoryBot.create_for_repository(:ephemera_folder, member_of_collection_ids: []) }
+    it 'tries to sort by title a nil array' do
+      expect(resource.decorate.collections.sort_by(&:title)).to be_truthy
```

Comment:
I don't think this test is testing what you expect. The above is pulling the variable @collections set by a controller, not calling collections on a decorator.